bot/code_review_bot/report/base.py

```diff
@@ -146,6 +146,9 @@ def pluralize(word, nb):
         else:
             comment = ""
 
+        if any(revision.contains(issue) for issue in issues):
+            comment = comment.replace(":", " (defects might be in the parent stack):")
```

Comment:
You should just add the content to comment:

```python
comment += " (defects might be in the parent stack
```

Reply:
I want to modify it like that, but a new line(`\n`) occurs.
![iss_533_newline](https://user-images.githubusercontent.com/29684524/79088043-61e47400-7d7c-11ea-874a-43873f066591.png)
If you modify it like a review, it goes down a line.
Is that okay?